docs/reference/query-dsl/match-query.asciidoc

```diff
@@ -157,15 +157,3 @@ The example above creates a boolean query:
 that matches documents with the term `ny` or the conjunction `new AND york`.
 By default the parameter `auto_generate_synonyms_phrase_query` is set to `true`.
 
-
-.Comparison to query_string / field
-**************************************************
-
```

Comment:
Reworded and moved to `query-string-query.asciidoc`:
https://github.com/elastic/elasticsearch/pull/45296/files#diff-d1da0885c6d0812eedd305523174ba5cR19